packages/launcher/src/renderer/launcher/onboarding/OnboardIdentityView.js

```diff
@@ -134,7 +143,12 @@ export default class OnboardIdentityView extends Component<Props, State> {
                 />
               </Column>
               <Column>
-                <Switch label="Make my name discoverable" name="discoverable" />
+                <Switch
+                  defaultValue={true}
```

Comment:
I don't think this switch should be on by default

Reply:
I feel like most people won't mind having a public profile and a lot will overlook this setting during onboarding. If we have it false by default I think majority of users will end up having no name and will be a bit of a poor experience?

Reply:
"discoverable" is a bit of a misnomer in any case. To me it implies being part of a searchable index, whereas in reality you would have to work pretty hard to "discover" anyone, let alone the person you're actually looking for

Reply:
Yea good point should probably use some better messaging there

Reply:
I agree, all of this has been rushed in and I don't think it's a good UX. I just think if we're saying Mainframe is "private by default", that's what our defaults should be, or at least we should make sure users are properly informed before publishing any data. But that's fine we don't have time anyways so let's just push what we have.